### ISSUE TITLE
jackson-jaxrs-json-provider dependency is obsolete?

### ISSUE THREAD

**Issue by KaVeKa** [p2-inc/phasetwo-java#15]
Hi there,

We are using this java client, but bumped into issues because of the dependency 
```
    <dependency>
      <groupId>com.fasterxml.jackson.jaxrs</groupId>
      <artifactId>jackson-jaxrs-json-provider</artifactId>
      <version>${jackson-version}</version>
    </dependency>
```

The exception we were getting was `java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlElement`.

Isn't this dependency obsolete since #7 , where you switched from the javax to the jakarta dependencies?

To fix our problems, we excluded the jackson-jaxrs-json-provider dependency in our pom from the phasetwo-admin-client.

Kind regards,
Kevin

**Comment by xgp** [comment]
@KaVeKa Thank you for pointing this out. Would you mind making a PR of the fix that worked for you?

**Comment by xgp** [comment]
fixed in #16
